docs/decorators.rst

```diff
@@ -82,6 +82,28 @@ The arguments and values are the same as ``@csp_update``::
         return render(...)
 
 
+``@csp_clear``
+================
+
+The ``@csp_clear`` decorator allows you to **clear** a CSP directive. Clearing
+a directive will take priority over any of the other options.
```

Comment:
I wonder if an admonition directive here like `warning` or even `danger` might be worthwhile.